spec/views/login_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Devise forms', type: :system do
```

Comment:
The type `system` is provided from this [tutorial](https://www.codewithjason.com/rails-testing-hello-world-using-rspec-capybara/), provided from Microverse exercise.
Besides, most of the test doesn't work if you put `feature` instead of `system`

Reply:
- [ ] Please address this you should use `feature` type as a requirement it normally works.. check an error you having and fix that this is an example:
`RSpec.describe 'Login page', js: true, type: :feature do
  describe 'Login Page requirements' do
    it 'has username and password inputs, and Log in button' do
      visit new_user_session_path
      expect(page).to have_field('user_email')
      expect(page).to have_field('user_password')
      expect(page).to have_button('Log in')
    end
  end`

Reply:
Did you test my spec files with `:feature` on? Did it work? On my end only `system` works.
Besides, this is from their official github getting started, so either of them is correct depending of which type of test is being written.
![image](https://user-images.githubusercontent.com/13990675/151559806-96875d22-e03c-4fe2-b105-196b63753b51.png)

Reply:
@lfmnovaes I am following requirements they want you guys to use `feature` which error are you getting?

Reply:
test